easycv/datasets/classification/pipelines/transform.py

```diff
@@ -134,3 +135,35 @@ def __repr__(self):
         repr_str += f'fill_color={self.fill_color}, '
         repr_str += f'fill_std={self.fill_std})'
         return repr_str
+
+
+@PIPELINES.register_module()
+class SimpleRandomCrop(object):
```

Comment:
we have already register `RandomCrop` from torchvision, please refer to: easycv/datasets/shared/pipelines/third_transforms_wrapper.py  and `MMRandomCrop` in `easycv/datasets/detection/pipelines/mm_transforms.py`

Reply:
done.
Remove simplerandomcrop.